src/library_nodefs.js

```diff
@@ -1,5 +1,5 @@
 mergeInto(LibraryManager.library, {
-  $NODEFS__deps: ['$FS', '$PATH'],
+  $NODEFS__deps: ['$PATH'],
```

Comment:
Yeah, using NODEFS or TTY requires FS, it's not possible otherwise.

So maybe the best thing is to do as you suggest in this PR, just remove deps from the fses to FS, and rely on FS being included when they are automatically anyhow. for that, we should probably also remove the FS dep from the other fses (even ones without postsets; for consistency), and we should write a comment in each of them instead of the deps, explaining this.

That seems ok to me. @juj, what do you think?